packages/bddapron/bddapron.2.3.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "Nicolas Berthier <m@nberth.space>"
+authors: ["Bertrand Jeannet"]
+homepage: "https://www.inrialpes.fr/pop-art/people/bjeannet/bjeannet-forge/bddapron/index.html"
+license: "LGPL-2.1"
+build: [
+  ["./configure" "--docdir" doc]
+  [make "JOBS=%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "bddapron"]
+  ["rm" "-r" "-f" "%{bddapron:doc}%"]
+]
+tags: [ "flags:light-uninstall" ]
+depends: [
+  "ocamlfind"
```

Comment:
It seems likely that the `build` tag should be here as well.

Reply:
I checked manually and there is no use of the `Findlib` library. Edited.